### PR TITLE
ethernet: enc28j60: issue in interfacing enc28j60 to pca10056

### PR DESCRIPTION
**Pull request by ngiriprasad**
Signed-off-by: Giri <giriprasad@semiconsoul.com>

### CODE REVIEW

File: soc/arm/nordic_nrf/nrf52/dts_fixup.h [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 /* SoC level DTS fixup file */
 
+#define DT_INST_0_MICROCHIP_ENC28J60_LOCAL_MAC_ADDRESS  	{0x12,0x45,0xcd,0x92,0x89,0xae}
```
Comment (galak) [inline soc/arm/nordic_nrf/nrf52/dts_fixup.h]: this should be coming from a dts node, not hand coded.

**Review by galak — CHANGES_REQUESTED** [review]
dts fixup should be coming from a .dts node.

**Comment by jfischer-phytec-iot** [comment]
@galak this PR is a reference for the issue #19398. I see no reason not to close it.

**Comment by jfischer-phytec-iot** [comment]
@ngiriprasad This PR does not make sense, you could just refer to your branch as well.

**Pull request closed without merging**